features/subscriber_can_comment_on_article.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Subscriber can comment on article
+
+    As a Subscriber
+    In order to voice my opinion or ask questions
+    I would like to be able to leave a comment on an article
+
+    Background:
+        Given the following user exists
+            | name    | email             | password | role   |
+            | Bill    | bill@mail.com     | password | member |
+            | Johanna | johanna@email.com | password | member |
+
+        And the following article exists:
+            | title | description    | content               | user_id | category_id | published |
+            | News  | This is a news | I like to eat cheese! | Bill    | Weather     | true      |
+
+        And I visit the landing page
+
+    Scenario: Logged in user can submit a comment [Happy Path]
```

Comment:
Should this be said in first person IE: 'When I am logged in...'?

Reply:
This is a Scenario description and not a step. But agree, will add an "A" to make it more readable.